bin/seed.js

```diff
@@ -0,0 +1,100 @@
+const mongoose = require('mongoose')
+const Movie = require('../models/Movie.model')
+
+const DB_NAME = 'express-cinema-dev'
+
+mongoose.connect(`mongodb://localhost/${DB_NAME}`, {
```

Comment:
Podemos requerir directamente de nuestra carpeta configs la configuración de la base de datos 😉 
```suggestion
require('../configs/db.config');
```